src/main/java/com/juniorstart/juniorstart/model/JobOffer.java

```diff
@@ -0,0 +1,56 @@
+package com.juniorstart.juniorstart.model;
+
+
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+
+import javax.persistence.OneToOne;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class JobOffer {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+	
+	@NotNull
+	private String message;
+	
+	@NotNull
+	private String type;
+	
+	@NotNull
+	private String position;
+	
+	@NotNull
+	private String technologies;
+	
+	@NotNull
+	private String requirements;
+	
+	@NotNull
+	private String contact;
+	
+	@OneToOne(fetch = FetchType.LAZY, optional = true)
+    @JoinColumn(name = "user_id", nullable = true)
+	@JsonIgnore
+    private User user;
```

Comment:
Tego usera można nazwać w stylu offerCreator albo coś w tym stylu

Reply:
to samo